system/Debug/Exceptions.php

```diff
@@ -141,23 +141,21 @@ public function exceptionHandler(Throwable $exception)
     }
 
     /**
-     * Even in PHP7, some errors make it through to the errorHandler, so
-     * convert these to Exceptions and let the exception handler log it and
-     * display it.
+     * The callback to be registered to `set_error_handler()`.
      *
-     * This seems to be primarily when a user triggers it with trigger_error().
+     * @return bool
      *
      * @throws ErrorException
      *
      * @codeCoverageIgnore
      */
     public function errorHandler(int $severity, string $message, ?string $file = null, ?int $line = null)
     {
-        if (! (error_reporting() & $severity)) {
-            return;
+        if (error_reporting() & $severity) {
+            throw new ErrorException($message, 0, $severity, $file, $line);
         }
 
-        throw new ErrorException($message, 0, $severity, $file, $line);
+        return false; // return false so that the error can propagate to other registered handlers
```

Comment:
Other registered handlers?
Isn't it the standard PHP error handler?

Reply:
In my understanding you can register a number of custom error handlers aside from PHP standard error handler because `set_error_handler()` returns the previously defined handler.
> Returns the previously defined error handler (if any). If the built-in error handler is used null is returned. If the previous error handler was a class method, this function will return an indexed array with the class and the method name.

Reply:
I thought we can change it, but it is not stackable. But I'm not sure.

Reply:
Let me check on that.

Reply:
The manual sample code seems to suggest it will fallback to the PHP handler depending on the return value:
```PHP
    /* Don't execute PHP internal error handler */
    return true;
```

Reply:
These comments seem to suggest stacking handlers:
* https://www.php.net/manual/en/function.set-error-handler.php#122518
* https://www.php.net/manual/en/function.set-error-handler.php#48933

Reply:
It seems it can be stacked.
https://github.com/symfony/symfony/blob/8435fb9362f3fa55790777e7f712548b25c8cc6f/src/Symfony/Component/ErrorHandler/ErrorHandler.php#L597-L599

Reply:
It is stackable that you can register any number of custom handlers. However when the actual error occurs, the last in the stack will be executed. If `false` is returned, the first in the stack (PHP standard handler) will be executed. If `true`, then it stops.

Reply:
Is it called stackable? 
It seems PHP has only one custom handler and it will be executed.
You can set any custom handler any number of times, but the registered one is the last one only.
Am I wrong?

Reply:
The PR made in php-src uses stacking as the term for it. But based on 3v4l.org, only the first and/or last registered handler gets executed.